src/main/java/net/openhft/chronicle/queue/impl/single/ThreadLocalAppender.java

```diff
@@ -0,0 +1,16 @@
+package net.openhft.chronicle.queue.impl.single;
+
+import net.openhft.chronicle.queue.ExcerptAppender;
+
+public class ThreadLocalAppender {
+
+    /**
+     * A thread local appender for the SingleChronicleQueue, this will give you the same thread local instance per thread.
+     *
```

Comment:
we need the javadoc from `acquireAppender` copied in here with all the reasons and caveats that you need to be aware of when using this